cluster/examples/openshift-bridge-marker.yml

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
What is the difference between kubernetes- and openshift-? In OVS we needed it because of ovs-vsctl dependency.

Reply:
Done